sql/dql/intro-joins/relative-database-intro.md

```diff
@@ -0,0 +1,106 @@
+---
+author: meridian
+
+type: normal
+
+category: must-know
+
+tags:
+  - introduction
+  - joins
+practiceQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+revisionQuestion:
+  formats:
+    - fill-in-the-gap
+  context: standalone
+---
+
+# Relational Databases
+
+---
+
+## Content
+
+You are now familiar with querying individual tables.
+
+Real databases are often comprised of more than one table. 
+
+Relational databases consist of tables and entries that connected to one another in some way. We call this connection a *relationship*.
+
+Consider the `pokedex` database, it contains two tables: the `pokemon_health` and `pokemon_attack` tables.
```

Comment:
```suggestion
Consider the `pokedex` database, it contains two tables: `pokemon_health` and `pokemon_attack`.
```

I think since we've already mentioned there are two tables, there is no need to say it again at the end of the sentence